jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -179,8 +179,6 @@ ecma_op_general_object_delete (ecma_object_t *obj_p, /**< the object */
     /* 5. */
     return ECMA_VALUE_FALSE;
   }
-
-  JERRY_UNREACHABLE ();
```

Comment:
Is it ok to remove this?

Reply:
Yes, above this line is an if/else if/else block with all branches returning. If we'd ever get here that would not be a program design flaw but a compiler error.

Reply:
I would remove the last `else` statement then, and simply do the return.